torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3400,6 +3580,10 @@ def evaluate_expr(self, orig_expr: "sympy.Expr", hint=None, fx_node=None):
                 eql, _ = self.create_fx_call_function(operator.eq, (fx_node, concrete_val))
                 node, fresh = self.create_fx_call_function(torch._assert, (eql,))
 
+            assert node is not None
+            if fresh:
+                node.meta["event"] = len(self.events) - 1
```

Comment:
what's this?

Reply:
This is so we know, when bisecting, what event corresponds to a given node. Say we want to replay until FX node `n`. This tells us until what event index we should replay to get to `n`.